tests/integration/config/test.yaml

```
@@ -0,0 +1,93 @@
+        heads: [4096, 2048, 1] # Note: Change to [4096, 2048, 1] for Task 2 and [4096, 2048, 512, 256, 1] Task 3
+
+    postprocessing:
+      metrics:
+        pred:
+          - "$lambda x: torch.sigmoid(x)" # Note: Change to $lambda x: torch.sigmoid(x) for Task 2 and Task 3
+        target:  # Note: Uncomment for Task 2 and Task 3
+          - "$lambda x: x.long()"
+      criterion: # Note: Uncomment for Task 2 and Task 3
+        pred:
+          - "$lambda x: x.squeeze(1)"
+        target: 
```

yamllint

[error] trailing spaces

(trailing-spaces)